File: Stack/StackDemo.java
import java.util.*;

class StackDemo
{
	public static void main(String args[])
	{
		try
		{
			Scanner sc = new Scanner(System.in);
			Stack<String> stack = new Stack();

			System.out.println("How many names in stack");
			int iNo = sc.nextInt();

			for(int i = 0;i < iNo; i++)
			{
				System.out.println("Enter "+(i+1)+" name ");
				String sName = sc.next();

				stack.push(sName);
			}
			System.out.println(stack);

			System.out.println("First name in stack is "+stack.peek());
			System.out.println("Pop name in stack is "+stack.pop());

			System.out.println("Enter name to search in stack");
			String name = sc.next();

			int ipos = stack.search(name);

			if(ipos == -1)
			{
				System.out.println("not found");
			}
			else
			{
				System.out.println("found at "+ipos);
			}

			System.out.println("Names in the Stack "+stack);
			Iterator itr = stack.iterator();
			while(itr.hasNext())
			{
				System.out.println(itr.next());
			}

			ArrayList AL = new ArrayList(stack);
			System.out.println("Names in the Array List "+AL);

			LinkedList list = new LinkedList(stack);
			System.out.println("Names in the Linked List "+list);

			Vector vector = new Vector(stack);
			System.out.println("Names in the Vector "+vector);

			stack.clear();
		}
		catch(Exception e)
		{
			System.out.println(e);
		}
	}
}

File: Stack/StackDemo.cs
using System;
using System.Collections;
using System.Collections.Generic;

class StackDemo
{
	public static void Main(string[] args)
	{
		try
		{
			Stack<int> stack = new Stack<int>();

			Console.WriteLine("How many elements in stack");
			int iNo = int.Parse(Console.ReadLine());

			for(int i = 0;i < iNo;i++)
			{
				Console.WriteLine("Enter {0} element",(i+1));
				int iN = int.Parse(Console.ReadLine());

				stack.Push(iN);
			}

			int size = stack.Count;

			Console.WriteLine("Total number of elements in stack are "+size);

			Console.Write("Stack elements are \t");
			foreach(int i in stack)
			{
				Console.Write(i+"\t");
			}
			Console.WriteLine();

			Console.WriteLine("Top most element in stack is "+stack.Peek());

			Console.WriteLine("pop element is "+stack.Pop());

			size = stack.Count;

			Console.WriteLine("Total number of elements in stack are "+size);

			Console.Write("Stack elements are \t");
			foreach(int i in stack)
			{
				Console.Write(i+"\t");
			}
			Console.WriteLine();
		}
		catch(Exception e)
		{
			Console.WriteLine(e);
		}
	}
}
